routers/repo/blame.go

```diff
@@ -119,8 +129,13 @@ func RefBlame(ctx *context.Context) {
 	ctx.Data["IsBlame"] = true
 
 	if ctx.Repo.CanEnableEditor() {
-		ctx.Data["CanDeleteFile"] = true
-		ctx.Data["DeleteFileTooltip"] = ctx.Tr("repo.editor.delete_this_file")
+		if isLFSLock {
```

Comment:
just use this condition here:
```suggestion
		if lfsLock != nil && lfsLock.OwnerID != ctx.User.ID {
```

Reply:
Done.